IPython/core/magics/execution.py

```diff
@@ -71,23 +72,45 @@ class TimeitResult(object):
 
     """
 
-    def __init__(self, loops, repeat, best, worst, all_runs, compile_time, precision):
+    def __init__(self, loops, repeat, all_runs, compile_time, precision):
```

Comment:
The timeit magic can return the result, removing `best` and `worst` attributes will break some code that rely on this. I doubt there are much code using this though.